react/week1/my-app/src/App.js

```diff
@@ -1,32 +1,25 @@
-import react, {useState} from 'react';
-import ToDoList from './ToDoList'
+import React, { useState } from "react";
+import TodoList from "./TodoList";
+
 function App() {
+  const todoList = [
```

Comment:
When constants or variables like this are static and don't change it's good practice to keep them outside the function so it doesn't render every time but it's a minor thing!